src/markdown.cpp

```diff
@@ -3540,7 +3540,9 @@ void MarkdownOutlineParser::parseInput(const QCString &fileName,
           docs = docs.left(labelStartPos)+                     // part before label
                  newLabel+                                     // new label
                  docs.mid(labelEndPos,lineLen-labelEndPos-1)+  // part between orgLabel and \n
-                 "\\ilinebr @anchor "+orgLabel+"\n"+           // add original anchor
+                 "\\ilinebr @ianchor "+orgLabel+               // add original anchor
```

Comment:
is the `i` in `@ianchor` a typo?

Reply:
oops this file should not have landed into this PR (it was a start for #9023, next issue I probably work on).
I'll remove the file from the PR